### ISSUE TITLE
Code example for DI-engine trading environment 

### ISSUE THREAD

**Issue by chadee1x** [opendilab/DI-engine#546]
Hi,

Thank you for a great package!

I was wandering how to code the equivalent of the code below that would use DI-engine environment for trading (the one with more than just 2 discrete actions - https://github.com/opendilab/DI-engine/tree/main/dizoo/gym_anytrading/envs)? 

I was studying your documentation however no code examples include your version of trading environment. I was trying to use analogies, but without any access. I would appreciate if you could provide a code example. Thank you very much.

# Imports
import gym
import gym_anytrading
from stable_baselines.common.vec_env import DummyVecEnv
from stable_baselines import A2C
import numpy as np
import pandas as pd
from matplotlib import pyplot as plt

# Create the Environment
env = gym.make('stocks-v0', df=df, frame_bound=(5,100), window_size=5)

env_maker = lambda: gym.make('stocks-v0', df=df, frame_bound=(5,100), window_size=5)
env = DummyVecEnv([env_maker])

# Train the Model
model = A2C('MlpLstmPolicy', env, verbose=1) 
model.learn(total_timesteps=1000000)

# Evaluate the Model
observation = env.reset()
while True:
    action = env.action_space.sample()
    observation, reward, done, info = env.step(action)
    # env.render()
    if done:
        print("info:", info)
        break

# Plot the Results
plt.cla()
env.render_all()
plt.show()

**Comment by PaParaZz1** [comment]
You can refer to demo [here](https://github.com/opendilab/DI-engine/blob/main/dizoo/gym_anytrading/config/stocks_dqn_config.py#L94),  just modify config according to your demands and then execute `python3 -u stocks_dqn_config.py`.

If you want to deploy the trained agent like SB3, you can refer to this [demo](https://github.com/opendilab/DI-engine/blob/main/dizoo/classic_control/cartpole/entry/cartpole_c51_deploy.py) to add your own codes with DI-engine.

If you have any other quesetions, you can continue to comment in this issue.

**Comment by chadee1x** [comment]
Thank you! After few setbacks, I managed to get the demo working in google colab on sample google stock data. However I faced a few other challenges i.e. how to:

1. Save the best model - it looks like during the training the best model is saved in:  ./stocks_dqn_seed0_221125_043003/ckpt/iteration_0.pth.tar 
Is this the best model?

2. Load the saved model
3. Evaluate loaded model on out of sample data
4. How to specify fees
5. Normalize the reward and features for training and use it in the same way during evaluation
6. Use additional features (for example in addition to OHLC and Volume and Adj. Close the data frame/csv file would also have columns with several TA indicators such as MA, RSI etc.)
7. Specify the column name for pricing data (Now 'Close" or Adjusted close is used)
8. Access action space data and particular reward - for example to map actions (buy, sell, hold) onto a pricing data frame as an additional column.

I would appreciate any help that would make these working.
